.pylintrc

```diff
@@ -23,7 +28,33 @@ disable=
  old-style-class,
 # "R" Refactor recommendations
  too-few-public-methods,
-
+# do not warn about pylint checks being disabled through comments in the code
+ locally-disabled,
+# For some reason, pylint doesn't get right instance members, and some types could not be inferred
+# Eg: [E1103 maybe-no-member] Instance of 'RootLogger' has no 'task' member File: convert2rhel/main.py, line 60, in main
+# we have several logger task, and s good approach to disable here, instead of each line
+ maybe-no-member,
+# Disabling duplicate code message, since do not make sense to refactor right now all duplicates code we have
+ duplicate-code,
```

Comment:
It's not an error. I'd not disable it since the pylint checker will not fail on that.
I'm sure there will many many more refactoring/stylistic/convention messages. We can deal with them later, instead of just "hiding" them by disabling the error.
Also, right now we are not stressed by some delivery deadlines so we can find a time to do some refactoring here and there.